thirdeye-spi/src/main/java/ai/startree/thirdeye/spi/Plugin.java

```diff
@@ -56,4 +59,8 @@ default Iterable<AnomalyPostProcessorFactory> getAnomalyPostProcessorFactories()
   default Iterable<EnumeratorFactory> getEnumeratorFactories() {
     return Collections.emptyList();
   }
+
+  default AccessControl getAccessControl(AccessControlConfiguration config) {
```

Comment:
nit: default should be just `AlwaysAllow`? or `null` and `AlwaysAllow` mean the same? just trying to simplify.

Reply:
This is the default method for all plugins, so null here indicates that the plugin does not provide access control. The other default methods return empty lists, but I don't think list makes sense for an access control plugin, so I chose null as the nothing here value.